_includes/footer.html

```diff
@@ -3,8 +3,6 @@
     <div class="footer-col">
       <h3>About</h3>
       <ul>
-        <li><a href="{{ '/about/' | prepend: site.baseurl }}">About TripleA</a></li>
```

Comment:
Should there still be a "History" link to the new forum post?  It seems it will be difficult to find, given that it's not a stickied post.  The history was the best part of the About page. :smile:

Reply:
At some point maybe, I think we need to evolve the history and make an updated thread for it. Agree history was best part, but it's about 13~15 years dated at this point. I think it was most interesting for developers, on website it's a tough call to prune it so we can focus on main mission of presenting the game and getting people to download it. The footer links I agree would be a good place for it, but again, in current form it's weak content : |

Reply:
Important to mention, website was intended to be a repo of static content and help information, to remove that content from forum posts. This does not seem to have worked out very well, so we are moving back the other way. Forums is where the information is best kept right now. Given that, we also don't want website to just be a table of links to the forum content. So I'm trying to refocus website here to have a cleaner mission of presenting the game to new users for download. We do have the forums link in header, so if someone does goes spelunking in forums they'll find the content easily enough I hope.